### ISSUE TITLE
Add more explanation to individual README

### ISSUE THREAD

**Issue by Himanshu40** [Himanshu40/Sorting-Algorithm#12]
Some points that can be added or can be changed:-
1. Pointing a typo and correcting it.
2. Adding compilation or interpretation process in each directory

**Comment by Jaidev2001** [comment]
Can you briefly explain me what to do ?
Compilation process like one by one explaining all the lines of code :thinking: :thinking:

**Comment by Himanshu40** [comment]
No, not like explaining code. It's like adding a compilation process for each programing language.
